src/main/java/at/hannibal2/skyhanni/config/features/Garden.java

```diff
@@ -625,6 +631,12 @@ public static class CustomSpeed {
     @ConfigAccordionId(id = 13)
     public int moneyPerHourShowOnlyBest = 5;
 
+    @Expose
+    @ConfigOption(name = "Expand top list by current Crop", desc = "Add active crop to the end of the list by expanding the set limit")
```

Comment:
Furthermore, this wording in the description sounds to me like with this toggle set to false, you don't see the current crop at all, or with it enabled, the current crop will always stay at the end of the list. Maybe find a better explanation to avoid confusing users and causing more support cases.